test/test_common/environment.cc

```diff
@@ -49,6 +48,30 @@ void TestEnvironment::initializeOptions(int argc, char** argv) {
   argv_ = argv;
 }
 
+bool TestEnvironment::isTestIpVersionOnly(const Network::Address::IpVersion& type) {
+  const char* option = ::getenv("IP_TEST_TYPE");
+  if (option == nullptr) {
+    return false;
+  }
+  if ((type == Network::Address::IpVersion::v4 && strcmp(option, "v4only") == 0) ||
+      (type == Network::Address::IpVersion::v6 && strcmp(option, "v6only") == 0)) {
+    return true;
+  }
+  return false;
+}
+
+std::vector<Network::Address::IpVersion> TestEnvironment::getIpTestParameters() {
+  std::vector<Network::Address::IpVersion> parameters;
+  if (TestEnvironment::isTestIpVersionOnly(Network::Address::IpVersion::v6) == false) {
```

Comment:
So, with the above suggestion of renaming, this could become:

```c++
if (TestEnvironment::shouldRunTestForIpVersion(Network::Address::IpVersion::v4)) {
  parameters.push_back(Network::Address::IpVersion::v4);
}
```